src/types/spot.ts

```diff
@@ -162,6 +162,19 @@ export interface WithdrawParams {
   amount: number;
   transactionFeeFlag?: boolean;
   name?: string;
+  walletType?: number;
+}
+
+export interface transferBrokerSubAccountParams {
+  fromId?: string;
+  toId?: string;
+  clientTranId?: string;
+  asset: string;
+  amount: number;
+}
+
+export interface transferBrokerSubAccount {
+  txnId: numberInString;
```

Comment:
```suggestion
export interface TransferBrokerSubAccountParams {
  fromId?: string;
  toId?: string;
  clientTranId?: string;
  asset: string;
  amount: number;
}

export interface TransferBrokerSubAccount {
  txnId: numberInString;
```

Sorry to be annoying, for the interface, please make the first letter upper case (like the other interfaces). It's only methods where the first letter is lower case.

Can you also bump the package.json version? It will make it easier to release it once I merge it. Thanks for the PRs!

Reply:
Thanks! This is what I meant with bumping the package version (increase it by 1, since it's a minor change):
https://github.com/tiagosiebler/binance/pull/272/files

It makes the release process easier for me to get this change onto the npm module.